src/doc/html_templates/html_templates.nit

```diff
@@ -196,3 +196,71 @@ class DocTopMenu
 		addn "</nav>"
 	end
 end
+
+redef class DocComposite
```

Comment:
You mean the doc from the intro? https://github.com/privat/nit/blob/master/src/doc/doc_base.nit#L68